lib/constructs/common/lambda-functions.ts

```diff
@@ -302,10 +304,14 @@ export class TimetableFunctions extends cdk.Construct {
 export class SyllabusFunctions extends cdk.Construct {
 
     readonly getFunction: Function;
+    readonly updateFunction: Function;
+    readonly SyllabusBucket: s3.Bucket;
```

Comment:
Attribute Name: Use camel case

```suggestion
    readonly syllabusBucket: s3.Bucket;
```

Reply:
Fix.